api/managementpb/checks.proto

```diff
@@ -77,8 +110,66 @@ message ChangeSecurityChecksRequest {
 
 message ChangeSecurityChecksResponse {}
 
+message ListFailedServicesRequest {}
```

Comment:
Should not result be paginated?

Reply:
Yeah, we figured since this is meant to be a summary (failed services and number of failed checks), it makes sense to display them at once.

Reply:
What if someone adds 100 or 300 services? All of them will be shown on single page?

Reply:
Thinking of it now, another approach could be to ignore the cache and query alertmanager directly for the results for this service.